### DISCUSSION TITLE
curl probes on 2.7.3

### DISCUSSION THREAD

**Discussion by PawanShirke** [oetiker/SmokePing#421] [Q&A]
Hi There,
I want to enable curl probe test with exiting fping tests running on 2.7.3 master and slave config. Is this a version limitation?

Here is my configurations:
**** Targets****

Targets:
probe = curl
menu: DockerHUB
title: Docker regitry
host: registry-1.docker.io

***** Probes *****
+ Curl

> probe-specific variables
binary = /usr/bin/curl
forks = 5
offset = 10%
step = 300
timeout = 20

> default for this target-specific variable
urlformat = https://%host/

**Comment by Joshua2504** [comment]
Hey, can you provide the correctly formatted Probes config?

**Comment by PawanShirke** [comment]
Certainly! Here it is:

```
*** Probes ***

+ FPing

binary = /usr/bin/fping

+ FPing6

binary = /usr/bin/fping6

+ Curl

# probe-specific variables
binary = /usr/bin/curl
forks = 5
offset = 10%
step = 300
timeout = 20

# a default for this target-specific variable
urlformat = https://%host/

```

**Comment by PawanShirke** [comment]
Any suggestion on this?
